api/dns.go

```diff
@@ -21,6 +21,6 @@ func (d *DNSHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	logger := loggerFromContext(r.Context())
 	log(logger, "Performing DNS")
 
-	Write(w, http.StatusOK, `{"domains": ["test.example.test", "test2.example.test"]}`)
+	Write(w, http.StatusOK, `{"domains": ["mystack.com"]}`)
```

Comment:
not sure what you want to do with that... here we should return a list containing all the custom domains that the user has linked to apps